### DISCUSSION TITLE
Changing border thickness of "MaterialDesignOutlinedDatePicker"

### DISCUSSION THREAD

**Discussion by washburnit** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2727] [Q&A]
HI,

how could I change the border thickness of the control in the subject?
I've set the control as follows:

`<DatePicker Name="dtLavo" Width="180" Style="{StaticResource MaterialDesignOutlinedDatePicker}" HorizontalAlignment="Left" Margin="20" mdix:HintAssist.Hint="Data Creazione"  FontFamily="Century Gothic" FontSize="15" BorderThickness="2"/>    `

but is seems that it has no effect (probably I'm not doing in the correct way).
Thank you, regards.

Roberto

**Comment by ElieTaillard** [comment]
Looks like a bug. Please, open a new issue about this.

**Comment by washburnit** [comment]
Done it.

Roberto
